src/App.tsx

```diff
@@ -1,61 +1,118 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/';
+import { TodoList } from './components/TodoList';
+
+const getTodos: Todo[] = todosFromServer.map(todo => ({
```

Comment:
```suggestion
const todoList: Todo[] = todosFromServer.map(todo => ({
```
It's not a function so the name is confusing